Project-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```diff
@@ -1 +1,45 @@
 /* All styles for your header Components live here */
+
+.lambdaLogo {
+    /* border: solid blue;
+    border-left: solid blue;
+    border-right: solid blue; */
+    width: 70%;
+    height: 10px;
+}
+
+.header {
+    color: rgb(67, 63, 63);
+    font-size: .5rem;
```

Comment:
Awesome work using rems for font size! Note that this will be pretty small, about 8px on default browser settings